contributed_definitions/nyaml/NXopt.yaml

```diff
@@ -406,18 +720,27 @@ NXopt(NXobject):
         described in ENTRY/INSTRUMENT/sample_stage.
       sample_name:
         doc: |
-          Descriptive name of the sample
-      sample_type:
+          Name of the sample, which is uaually pleases human readability.
+          This may be, but does not have to be identical to the sample_id.
+      sample_id:
+        exists: recommended
         doc: |
-          Specify the type of sample, e.g. thin film, single crystal etc.
-        enumeration: [thin film, single crystal, poly crystal, single layer, multi layer]
-      layer_structure:
+          Uniquely identifying name of the sample.
+      physical_form:
+        exists: recommended
         doc: |
-          Qualitative description of the layer structure for the sample,
-          starting with the top layer (i.e. the one on the front surface, on
-          which the light incident), e.g. native oxide/bulk substrate, or
-          Si/native oxide/thermal oxide/polymer/peptide.
+          State the form of the sample, examples are:
+          thin film, single crystal, poly crystal, amorphous, single layer,
+          multi layer, liquid, gas, pellet, powder.
+      description:
+        exists: optional
+        doc: |
+          Free text description of the sample.
+      situation:
```

Comment:
I recommend not introducing `situation `and rather describe the sample situation by `NXenvironments`, we want to get rid of it in MPES as well.

Reply:
okay, good to hear. Just thaught, that it would be good to follow this as it is as well in NXsample from NIAC.

Is it just the location of situation or as well the selections? (did not found anything which would represent this in NXenvironment)

Reply:
I would get rid of `situation` and just use multiple NXenvironments for describing the surrounding pressure, temperature, etc. You can have a look how it is currently implemented in [NXmpes](https://fairmat-nfdi.github.io/nexus_definitions/classes/contributed_definitions/NXmpes.html#nxmpes-entry-sample-temperature-group). You can link sensors from the instrument within that sample environment.

Reply:
Hm okay, it is as well designed in this way - But the "sample situation" contains significantly more than that, as it is aside of pressure, as well considering the chemical nature of the surroundings. 
Of course, this is usually not measured/verified but assumed from the experimental setup. I think a free text description has still valueable information and shifted it towards NXenvironment with the name "sample_medium".